src/HeaderSecurity.php

```diff
@@ -129,7 +129,7 @@ public static function isValid($value)
     public static function assertValid($value)
     {
         if (! self::isValid($value)) {
-            throw new InvalidArgumentException('Invalid header value');
+            throw new InvalidArgumentException("'$value' is not valid header value");
```

Comment:
We typically use `sprintf()` for interpolation in exception messages; I'll make that change on merge.